debian/ubuntu-advantage-tools.postinst

```diff
@@ -16,6 +16,8 @@ fi
 # https://lintian.debian.org/tags/postinst-does-not-load-confmodule.html
 . /usr/share/debconf/confmodule
 
+PYTHON="/usr/bin/env PYTHONPATH=/usr/lib/python3/dist-packages /usr/bin/python3"
```

Comment:
Can we only have `/usr/lib/python3/`  as the python path here ?

Reply:
I just tested and unfortunately, having only `/usr/lib/python3/` doesn't work here